content/200-concepts/100-components/01-prisma-schema/07-names-in-underlying-database.mdx

```diff
@@ -17,19 +17,23 @@ The [Prisma schema](./) includes mechanisms that allow you to define names of ce
 
 ## Mapping collection/table and field/column names
 
-Sometimes the names used to describe entities in your database might not match the names you would prefer in your generated API. A common approach for naming tables/collections in databases for example is to use plural form and [snake_case](https://en.wikipedia.org/wiki/Snake_case) notation. Prisma on the other hand has recommended model [naming conventions (singular form, PascalCase)](/reference/api-reference/prisma-schema-reference#naming-conventions) <span class="api"></span> which differ from that. Mapping names in the Prisma schema allows you to influence the naming in your Client Api without having to change the underlying database names.
+Sometimes the names used to describe entities in your database might not match the names you would prefer in your generated API. Mapping names in the Prisma schema allows you to influence the naming in your Client Api without having to change the underlying database names.
+
+A common approach for naming tables/collections in databases for example is to use plural form and [snake_case](https://en.wikipedia.org/wiki/Snake_case) notation. Prisma on the other hand has recommended model [naming conventions (singular form, PascalCase)](/reference/api-reference/prisma-schema-reference#naming-conventions) <span class="api"></span> which differ from that. 
+
+`@map` and `@@map` allow you to [tune the shape of your Prisma Client API](../prisma-client/working-with-prismaclient/use-custom-model-and-field-names#using-map-and-map-to-rename-fields-and-models-in-the-prisma-client-api) by decoupling model and field names from table and column names in the underlying database.
 
 ### Map collection / table names
 
-For example, when you introspect a database with a table named `comments`, the resulting Prisma model will look like this:
+As an example, when you [introspect](./introspection) a database with a table named `comments`, the resulting Prisma model will look like this:
 
 ```prisma
 model comments {
   // Fields
 }
 ```
 
-However, you can still follow the naming convention without renaming the underlying `comments` table in the database by using the [`@@map`](/reference/api-reference/prisma-schema-reference#map-1) <span class="api"></span> attribute:
+However, you can still choose `Comment` as the name of the model (e.g. to follow the naming convention) without renaming the underlying `comments` table in the database by using the [`@@map`](/reference/api-reference/prisma-schema-reference#map-1) <span class="api"></span> attribute:
```

Comment:
deemphasize the naming convention, most use cases will just be "I like it better this way"